apis/Google.Cloud.Storage.V1/Google.Cloud.Storage.V1/StorageClient.CopyObject.cs

```diff
@@ -53,6 +53,12 @@ public abstract partial class StorageClient
         /// <param name="options">Additional options for the copy operation. May be null, in which case appropriate
         /// defaults will be used.</param>
         /// <param name="cancellationToken">The token to monitor for cancellation requests.</param>
+        /// <remarks>
+        /// The fact that <paramref name="destinationBucket"/> and <paramref name="destinationObjectName"/> are
+        /// optional is a mistake. These default value of null for these parameters is invalid for this method,
```

Comment:
Just read it again and I think that "The default value of null for these parameters is invalid for this method..." sounds better, so maybe `These` => `The`. Please ignore if this doesn't make sense and sorry for an afterthought.

Reply:
Whoops, yes, it was intended to be "The". Will include this change in a separate commit but on a PR which is already dealing with other Storage matters.